plugins/techdocs-node/api-report.md

```diff
@@ -89,7 +89,7 @@ export const getLocationForEntity: (
 ) => ParsedLocationAnnotation;
 
 // @public
-export const getMkDocsYml: (
+export const getMkdocsYml: (
```

Comment:
I believe you have to deprecate this and export a new one, otherwise, it would be a breaking change.

Reply:
Good catch. Thank you.